src/sentry/runner/commands/backup.py

```diff
@@ -385,6 +385,7 @@ def load_data(
                 input: IO[bytes] = BytesIO(decrypt_encrypted_tarball(src, decryptor))
             except DecryptionError as e:
                 click.echo(f"Invalid {side.name} side tarball: {str(e)}", err=True)
+                raise
```

Comment:
ah I see, the `-> json.JSONData` annotation was misleading you here!  (it's `Any` by the way -- could probably do better with `dict[str, Any]` ?

Reply:
Changed the type annotation to your suggestion.